e2e/jest.test-setup.ts

```diff
@@ -17,43 +17,7 @@ beforeEach(async () => {
   // See https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetdefaultnavigationtimeouttimeout
   page.setDefaultNavigationTimeout(navTimeOut);
   page.setDefaultTimeout(timeOut);
-});
-
-/**
- * At the end of each test completion, do:
- * - Disable network interception.
- * - Delete broswer cookies.
- * - Reset global page and browser variables.
- */
-afterEach(async () => {
-  await page.deleteCookie(...await page.cookies());
-  await jestPuppeteer.resetPage();
-  await jestPuppeteer.resetBrowser();
-});
 
-/**
- * Enable network interception in new page and block unwanted requests.
- */
-beforeAll(async () => {
-  await page.setRequestInterception(true);
```

Comment:
`setRequestInterception(true)` interferes with login by cookies. It has to be removed.